tensorflow/python/compiler/tensorrt/test/testdata/gen_tftrt_model.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Saves a SavedModel after TensorRT conversion.
+   The saved model is loaded and executed by tests to ensure backward
+   compatibility across TF versions.
+   The script may not work in TF1.x.
+
+   Instructions on how to use this script:
+   - Execute the script as follows:
+       python gen_tftrt_model
+   - Rename tftrt_saved_model to what makes sense for your test.
+   - Delete directory tf_saved_model unless you want to use it.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
```

Comment:
There is an error when trying to submit this PR:
'import tensorflow' statements are no allowed inside TF's code base... Please import modules directly.

Also could you help to add a build rule for this file?

Reply:
This file is not supposed to be used by any test. It's supposed to be used manually by developers to generate a test model if needed. Do we still need to add it to BUILD?

Reply:
I think adding a build rule will make our life easier: we just need to run `bazel run` to get the model. But I don't have strong opinion. But at least replace `import tensorflow` with direct module imports.

Reply:
I removed the import.

I also noticed the test model was missing in the data dependency of trt_convert_test which is fixed now. I missed this because I run tests without bazel :(

I haven't added gen_tftrt_model.py to BUILD. I would need to learn how to do that first. I suggest to postpone this for now.